src/Microsoft.Dnx.Compilation.CSharp/RoslynAnalyzationModule.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Reflection;
+using System.Runtime.Versioning;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.Dnx.Runtime;
+
+namespace Microsoft.Dnx.Compilation.CSharp
+{
+    public class RoslynAnalyzationModule
```

Comment:
I'd call this something like `AnalyzerSet`. The Roslyn prefix is redundant (and we should be removing it from other places) and `Module` is only used for `ICompileModule` to match up with that.